.github/workflows/integration-tests.yml

```diff
@@ -107,6 +73,29 @@ jobs:
             ccip_changes:
               - '**/*ccip*'
               - '**/*ccip*/**'
+      - name: Filter not ignored changes
+        uses: ./.github/actions/filter-changes
+        id: filter
+        with:
+          # Filter files that match **every** pattern in this list
+          every-filter: |
+            - '**'                     # Initially include all files
+            - '!**/*.md'               # Exclude all markdown files (lowercase)
+            - '!**/*.MD'               # Exclude all markdown files (uppercase)
+            - '!**/.github/**'         # Exclude all changes under the .github directory
+            # Exclude all Go test files, except those explicitly included in "any-filter"
+            - '!**/*_test.go'
+            - '!testdata/**'           # Exclude all testdata directories
+            - '!tools/**'              # Exclude all tools directories
+          # Filter files that match **any** of these patterns
+          any-filter: |
+            - 'integration-tests/**/*.go'                 # Include all go files in integration-tests
+            - 'integration-tests/**/*.toml'               # Include all toml files in integration-tests
+            - '.github/workflows/integration-tests.yml'   # Include the integration-tests workflow itself  
+            - '.github/actions/build-chainlink-image/**'
```

Comment:
See my comment at https://github.com/smartcontractkit/chainlink/pull/14585/files#diff-31bd198ce2c6956ed10bdfe25a233b755f3a201ad094a67c449115ef76d9ff43R75 on why this is a bad idea.